walletlibrary/src/main/java/com/microsoft/walletlibrary/VerifiedIdClientBuilder.kt

```diff
@@ -113,7 +127,8 @@ class VerifiedIdClientBuilder(private val context: Context) {
     private fun getUserAgent(applicationContext: Context): String {
         return try {
             val packageManager = applicationContext.packageManager
-            val applicationInfo = packageManager.getApplicationInfo(applicationContext.packageName, 0)
+            val applicationInfo =
```

Comment:
nit: was this line supposed to be altered?

Reply:
It is automatic indentation performed by IDE. It must be because of length of the line. This change is fine.